### PR TITLE
Created configuration for GitHub Actions.

### LINKED ISSUE #8: Add CI
**Summary.**
Add configuration for CI, could be using GitHub Actions.

### PR DESCRIPTION
**Pull request by juanpa097**
### Summary
Created `yml` file with action configuration.

### Test Plan
When making a PR or push to master the actions should be executed.

### Related Issues
closes #8

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    branches:
+      - master
+      - dev  
+
+jobs:
+  lint:
+    name: Static Code Analysis With Ktlint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (johan-smc) [inline .github/workflows/maven.yml]: This was the error

Comment (juanpa097) [reply .github/workflows/maven.yml]: Thats what happens when you copy and paste.

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (juanpa097) [inline .idea/misc.xml]: This file should be ignored

File: src/main/java/antlr/generate/Treepat.interp [hunk 3]
```diff
@@ -43,4 +43,4 @@ node
...
...2, 2, 46, 39, 3, 2, 2, 2, 47, 5, 3, 2, 2, 2, 48, 51, 5, 18, 10, 2, 49, 51, 5, 16, 9, 2, 50, 48, 3, 2, 2, 2, 50, 49, 3, 2, 2, 2, 51, 7, 3, 2, 2, 2, 52, 53, 7, 3, 2, 2, 53, 54, 7, 12, 2, 2, 54, 55, 5, 10, 6, 2, 55, 56, 7, 13, 2, 2, 56, 9, 3, 2, 2, 2, 57, 59, 5, 12, 7, 2, 58, 57, 3, 2, 2, 2, 59, 60, 3, 2, 2, 2, 60, 58, 3, 2, 2, 2, 60, 61, 3, 2, 2, 2, 61, 11, 3, 2, 2, 2, 62, 67, 5, 14, 8, 2, 63, 64, 7, 4, 2, 2, 64, 66, 5, 14, 8, 2, 65, 63, 3, 2, 2, 2, 66, 69, 3, 2, 2, 2, 67, 65, 3, 2, 2, 2, 67, 68, 3, 2, 2, 2, 68, 13, 3, 2, 2, 2, 69, 67, 3, 2, 2, 2, 70, 71, 5, 4, 3, 2, 71, 15, 3, 2, 2, 2, 72, 73, 7, 5, 2, 2, 73, 74, 5, 8, 5, 2, 74, 75, 7, 6, 2, 2, 75, 76, 7, 7, 2, 2, 76, 81, 3, 2, 2, 2, 77, 78, 5, 8, 5, 2, 78, 79, 7, 7, 2, 2, 79, 81, 3, 2, 2, 2, 80, 72, 3, 2, 2, 2, 80, 77, 3, 2, 2, 2, 81, 17, 3, 2, 2, 2, 82, 86, 5, 24, 13, 2, 83, 86, 5, 22, 12, 2, 84, 86, 5, 20, 11, 2, 85, 82, 3, 2, 2, 2, 85, 83, 3, 2, 2, 2, 85, 84, 3, 2, 2, 2, 86, 19, 3, 2, 2, 2, 87, 88, 7, 8, 2, 2, 88, 89, 5, 24, 13, 2, 89, 21, 3, 2, 2, 2, 90, 91, 5, 24, 13, 2, 91, 92, 7, 9, 2, 2, 92, 23, 3, 2, 2, 2, 93, 94, 5, 26, 14, 2, 94, 25, 3, 2, 2, 2, 95, 96, 7, 10, 2, 2, 96, 27, 3, 2, 2, 2, 10, 34, 43, 46, 50, 60, 67, 80, 85]
```
Comment (juanpa097) [inline src/main/java/antlr/generate/Treepat.interp]: I think we should ignore this file.

**Pull request merged**